### PR TITLE
Update Set-ADAccountPassword.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#450 
Action Taken

Added the following:
"The cmdlet will not be executed"
-"-WhatIf command works in Windows Server 2016 and the accompanying Remote Server Administration Tools (RSAT) package."

References:
Uservoice 1
This is fixed in Windows Server 2016 and the accompanying RSAT.
https://windowsserver.uservoice.com/forums/304621-active-directory/suggestions/10844361-set-adaccountpassword-whatif-does-not-work

Uservoice 2
https://windowsserver.uservoice.com/forums/304621-active-directory/suggestions/11130105-set-adaccountpassword-whatif-does-not-work-inst

-Tested the cmdlets in Windows Server 2008 R2 then checked the passwordlastset attribute, seen that it execute the password changes.
**_-Perhaps, we can possibly verify other affected versions._**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
I am ok but something like this could be better:

"The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes."

File: docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md [hunk 1]
```diff
@@ -412,8 +412,11 @@ Accept wildcard characters: False
 
 ### -WhatIf
 Shows what would happen if the cmdlet runs.
-The cmdlet is not run.
+The cmdlet will not be executed. 
 
+> [!NOTE]
+> -WhatIf command works in Windows Server 2016 and the accompanying Remote Server Administration Tools (RSAT) package.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md]:
```suggestion
> -The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes.  
```

**Comment by dariomws306** [comment]
sorry @JohanFreelancer9, I think "Shows what would happen if the cmdlet runs. The cmdlet will not be executed. " should be replaced with "The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes."

We should leave the note as well as it is.

**Comment by get-itips** [comment]
Agree with Dario, WhatIf parameter should be the same for any cmdlet.

File: docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md [hunk 2]
```diff
@@ -412,8 +412,11 @@ Accept wildcard characters: False
 
 ### -WhatIf
 Shows what would happen if the cmdlet runs.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md]:
```suggestion
```

File: docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md [hunk 3]
```diff
@@ -412,8 +412,11 @@ Accept wildcard characters: False
...
-The cmdlet is not run.
+The cmdlet will not be executed.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md]:
```suggestion
The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes. 
```

File: docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md [hunk 4]
```diff
@@ -412,8 +412,11 @@ Accept wildcard characters: False
 
 ### -WhatIf
 Shows what would happen if the cmdlet runs.
-The cmdlet is not run.
+The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes.
```
Comment (nenonix) [inline docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md]:
```suggestion
The WhatIf switch causes the command to simulate its results. Using this switch allows you to view changes that would occur, without having to commit those changes.
```

**Comment by tseward** [comment]
I'd change the NOTE to read:

-WhatIf command functionality was first fixed in the Windows 1607 RSAT package. If using an RSAT package prior to the Windows 1607 RSAT package, this parameter will not function.

**Comment by e0i** [comment]
@AndreaBarr This PR can now be merged. Thanks.

**Pull request merged**